examples/jsm/postprocessing/Pass.js

```diff
@@ -56,14 +56,12 @@ class FullscreenTriangleGeometry extends BufferGeometry {
 
 }
 
-const _geometry = new FullscreenTriangleGeometry();
 
 class FullScreenQuad {
 
 	constructor( material ) {
-
+		const _geometry = new FullscreenTriangleGeometry();
```

Comment:
The geometry should be shared across all instance of `FullScreenQuad`. 

Moving the line it inside the ctor will break this aspect.

Reply:
Why is it in issue if objects are created in module scope? This is patterns we use through the entire library code.

Reply:
Thank you for your answer. Maybe it's something else. After adding postprocessing to my project, I get the error "Cannot call the constructor of class BufferGeometry without 'new'" on the server side in next.js. I also bump versions of three.js from 0.158.3 to 0.162.0. 


I don't know why I suddenly have a problem. The project had been working properly for over 2 years and nothing was being run on the server side. The code only ran on the browser side.

Reply:
> "Cannot call the constructor of class BufferGeometry without 'new'"

Um, I don't understand how the removal of the above line relates to this error message. 

I suggest you use to [forum](https://discourse.threejs.org/) to find out the root cause of the regression. If it turns out to be an issue in the engine, we are of course open for a fix. But the current change the PR proposes is not acceptable, imo.

Reply:
FullscreenTriangleGeometry extends BufferGeometry and I have an error in that line ;)